airflow-core/src/airflow/api_fastapi/core_api/datamodels/task_instances.py

```diff
@@ -202,6 +202,7 @@ class ClearTaskInstancesBody(StrictBaseModel):
         description="(Experimental) Run on the latest bundle version of the dag after "
         "clearing the task instances.",
     )
+    is_running_message: bool = False
```

Comment:
We should probably change the naming here. "force", "force_running", "bypass_running" or something else, `message` makes me think of  a 'string'